nni/runtime/tuner_command_channel/shim.py

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+"""
+Compatibility layer for old protocol APIs.
+
+We are working on more semantic new APIs.
+"""
+
+from __future__ import annotations
+
+from .command_type import CommandType
+from .websocket import WebSocket
+
+_ws: WebSocket = None  # type: ignore
```

Comment:
we do not support running two experiments in one python process, right? as they share the same `_ws`.

Reply:
We do.
This file is the temporary compatibility layer to smooth the migration, not the final API.